plugins/out_file/file.c

```diff
@@ -97,6 +98,7 @@ static int cb_file_init(struct flb_output_instance *ins,
     ctx->delimiter = NULL;
     ctx->label_delimiter = NULL;
     ctx->template = NULL;
+    ctx->line_append = 0;
```

Comment:
Could you use `FLB_FALSE` ?

Reply:
OK, I will change it in next commit.
Forgive me for my unfamiliar about C language style.